app/src/main/java/com/sopt/now/presentation/ui/auth/signin/SignInViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.sopt.now.presentation.ui.auth.signin
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.sopt.now.data.User
+
+class SignInViewModel : ViewModel() {
+    private val _user = MutableLiveData<User?>()
+    val user: LiveData<User?> = _user
+
+    private val _signInState = MutableLiveData(false)
+    val signInState: LiveData<Boolean> = _signInState
```

Comment:
Kotlin 2.0을 쓴다면? explicit backing fields 를 활용할 수 있어욥

```suggestion
    val user: LiveData<User?>
        field = MutableLiveData<User?>()

    val signInState: LiveData<Boolean>
        field = MutableLiveData(false)
```

[관련 문서](https://medium.com/@omarsahl/kotlin-language-features-from-kotlinconf-2024-whats-new-and-what-s-next-a4668eae5e9d)